### ISSUE TITLE
Observability Addon Invalid IP Error

### ISSUE THREAD

**Issue by choyuansu** [canonical/microk8s-core-addons#104]
#### Summary

I'm trying to run `microk8s enable observability` but got the following error:

<code>
Error: UPGRADE FAILED: failed to create resource: Endpoints "kube-prom-stack-kube-prome-kube-controller-manager" is invalid: [subsets[0].addresses[0].ip: Invalid value: "192.168.0.11,192.168.0.22,192.168.0.33": must be a valid IP address, (e.g. 10.9.8.7 or 2001:db8::ffff), subsets[0].addresses[0].ip: Invalid value: "192.168.0.11,192.168.0.22,192.168.0.33": must be a valid IP address]
</code>

#### What Should Happen Instead?

Microk8s should install the observability addon without errors.

#### Reproduction Steps

1. `microk8s enable observability`

#### Introspection Report

[inspection-report-20220921_194101.tar.gz](https://github.com/canonical/microk8s-core-addons/files/9621301/inspection-report-20220921_194101.tar.gz)

#### Are you interested in contributing with a fix?

Yes.

**Comment by ktsakalozos** [comment]
Hi @choyuansu 

Thank you for reporting this. We have a patch released and is currently on the edge channel (`sudo snap install microk8s --classic --channel=1.25/edge`). It should be available on the 1.25/stable soon. For now you have the following options:

- install microk8s from the 1.25 channel
- manually apply the patch found in [1] by editing `/var/snap/microk8s/common/addons/core/addons/observability/enable`
- remove the core repository and re-add it pointing to the main branch of this repository
  ```
  sudo microk8s addons repo remove core
  sudo microk8s addons repo add core https://github.com/canonical/microk8s-core-addons
  ```

[1] https://github.com/canonical/microk8s-core-addons/pull/103/files

**Comment by choyuansu** [comment]
Thank you for the quick response! I'm able to enable the observability addon now.
